test/lenses.js

```diff
@@ -41,16 +56,24 @@ describe('view, over, and set', function() {
 
   they('may be applied to composed lenses', function() {
     var streetLens = R.compose(addressLens, headLens);
+    var dogLens = R.compose(R.lensPath(['pets']), R.lensPath(['dog']));
+
+    eq(R.view(dogLens, alice), R.view(R.lensPath(['pets', 'dog']), alice));
```

Comment:
But why aren't there any separate tests for lensIndex or lensProp ? The general idea of `lenses` seems to be to test lens interop.